clab/export.go

```diff
@@ -70,6 +70,11 @@ func (c *CLab) exportTopologyDataWithTemplate(w io.Writer, p string) error {
 
 	for _, n := range c.Nodes {
 		e.NodeConfigs[n.Config().ShortName] = n.Config()
+		gc, err := n.GetContainers(ctx)
+		if err != nil {
+			return err
+		}
```

Comment:
@steiler works nicely, this piece is the only thing that I am hesitant about.
Doing an API Get for all nodes just to fetch port bindings is not optimal:

1. not all nodes have port bindings defined
2. and even if they have them, maybe we can just update the binding information once we created the node? I wonder if we already receive the port binding info as a result of a container create?

Reply:
since you've now made it part of NodeConfig, we should be able to get this info earlier on as part of the deployment, yes.